sdk/attestation/azure-security-attestation/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0-beta.2 (2022-05-10)
 
 ### Features Added
```

Comment:
nit: I think we can remove this heading all together given we have no new features, and reduce the noise-to-signal ratio for customers reading the changelog. The single bullet here isn't really relevant to users.

Reply:
Also, this text seems redundant and can be removed, along with surrounding white space (in the breaking change section):
> Breaking Changes from API Review

Reply:
Maybe? The scripts get really annoyed if the various elements aren't all there. I'm concerned that removing the Features Added section is going to upset the release scripts.

And I wanted to express that all the breaking changes in this list are a consequence of API Review and not just me being completely random in my changes.

Reply:
> Maybe? The scripts get really annoyed if the various elements aren't all there. I'm concerned that removing the Features Added section is going to upset the release scripts.


The release scripts shouldn't care and be resilient to such headings not being there. We have done them in the past, without issues. For example, see:
https://github.com/Azure/azure-sdk-for-cpp/blob/main/sdk/core/azure-core/CHANGELOG.md#131-2021-11-05

> And I wanted to express that all the breaking changes in this list are a consequence of API Review and not just me being completely random in my changes.

The intended audience for the changelog are our customers. That detail shouldn't be relevant to them :)